dnf-behave-tests/features/steps/cmd.py

```diff
@@ -150,6 +150,18 @@ def given_enable_plugin(context, plugin):
         context.dnf["plugins"].append(plugin)
 
 
+@behave.given("I successfully execute dnf with args \"{args}\"")
+def given_i_successfully_execute_dnf_with_args(context, args):
+    when_I_execute_dnf_with_args(context, args)
+    then_the_exit_code_is(context, 0)
+
+
+@behave.given("I successfully execute rpm with args \"{args}\"")
+def given_i_successfully_execute_rpm_with_args(context, args):
+    when_I_execute_rpm_with_args(context, args)
+    then_the_exit_code_is(context, 0)
```

Comment:
I guess I'll change it... But I don't like the way Behave works in this regard...

We should just rename all step function names to `step_impl` then, to avoid the illusion the function name matters.